benchmarks/versus-external-pools/fixed-threadwork.js

```diff
@@ -11,7 +11,7 @@ const data = {
   taskType: process.env['TASK_TYPE']
 }
 
-const threadPool = new ThreadPool({ task: functionToBench, size: size })
+const threadPool = new ThreadPool({ task: functionToBench, size: Number(size) })
```

Comment:
**nitpick (non-blocking):** We could move the conversion to the assignment itself. So in this case we could move the `Number(...)` to line 7.  
Other files are also affected.

Reply:
Improper type might have serious side effect, especially string that should be number. So that's not nitpicking. I've made the change. Thanks.